command/rootchain/initcontracts/init_contracts.go

```diff
@@ -230,6 +237,26 @@ func deployContracts(outputter command.OutputFormatter, client *jsonrpc.Client,
 	manifest.RootchainConfig = rootchainConfig
 
 	for _, contract := range deployContracts {
+		if contract.name == "RootERC20" && params.rootERC20TokenAddr != "" {
```

Comment:
You could extract this before the `for` loop, and just don't add it to the `deployContracts` array if `params.rootERC20TokenAddr` is provided. WDYT?